src/aead/aes_gcm.rs

```diff
@@ -129,4 +127,56 @@ mod tests {
         test_aead(&aead::AES_256_GCM,
                   "crypto/cipher/test/aes_256_gcm_tests.txt");
     }
+
+    fn convert_block(data: &[u8]) -> [u8; AES_BLOCK_SIZE] {
+        let mut buf = [0u8; AES_BLOCK_SIZE];
+        buf.clone_from_slice(data);
+        buf
+    }
+
+    #[test]
+    pub fn test_aes() {
+        test::from_file("src/aead/aes_tests.txt", |section, test_case| {
+            assert_eq!(section, "");
+            let key = test_case.consume_bytes("Key");
+            let input = convert_block(&test_case.consume_bytes("Input"));
+            let output = convert_block(&test_case.consume_bytes("Output"));
+
+            let mut aes_key = AesKey {
+                rd_key: [0u32; 4 * (AES_MAX_ROUNDS + 1)],
+                rounds: 0,
+            };
+
+            unsafe {
+                // Key setup
+                if AES_set_encrypt_key(key.as_ptr(), key.len() * 8, &mut aes_key) != 0 {
+                    panic!("Failed to set aes key");
+                }
```

Comment:
You can just do `assert_eq!(GFp_AES_set_encrypt_key(...), 0, "GFp_AES_set_encrypt_key failed");`